orm/hibernate-orm-4/src/test/java/org/hibernate/bugs/ORMStandaloneJpaUnitTestCase.java

```diff
@@ -0,0 +1,203 @@
+/*
+ * Copyright 2014 JBoss Inc
```

Comment:
We also need to add a license.txt to this project as well. I realized it's missing.